autopush/main.py

```diff
@@ -209,6 +209,7 @@ def from_argparse(cls, ns):
             bear_hash_key=ns.auth_key,
             proxy_protocol_port=ns.proxy_protocol_port,
             use_cryptography=ns.use_cryptography,
+            enable_simplepush=ns.enable_simplepush,
```

Comment:
FYI this and use_cryptography could go in AutopushSettings.from_argparse since both endpoint/conn need it